<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4" rowstretch="0,0,1">
   <item row="0" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2">
    <widget class="QGroupBox" name="voltage_groupBox">
     <property name="title">
      <string>Voltage</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="2" column="0">
       <widget class="QLabel" name="set_label_3">
        <property name="text">
         <string>Set</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="actual_label_3">
        <property name="text">
         <string>Actual</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="set_voltage_doubleSpinBox"/>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="actual_voltage_doubleSpinBox"/>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="cv_mode_checkBox">
        <property name="text">
         <string>CV</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="current_groupBox">
     <property name="title">
      <string>Current</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1" colspan="2">
       <widget class="QCheckBox" name="cc_mode_checkBox">
        <property name="text">
         <string>CC</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="set_label">
        <property name="text">
         <string>Set</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="actual_label">
        <property name="text">
         <string>Actual</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QDoubleSpinBox" name="set_current_doubleSpinBox"/>
      </item>
      <item row="2" column="2">
       <widget class="QDoubleSpinBox" name="actual_current_doubleSpinBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>Live</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QGroupBox" name="plot_groupBox">
     <property name="title">
      <string>History</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout"/>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="device_name_label">
     <property name="text">
      <string>DEVICE</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="history_len_doubleSpinBox"/>
   </item>
   <item row="1" column="2">
    <widget class="QDoubleSpinBox" name="poll_time_doubleSpinBox"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
